src/components/Chat/ChatItem.tsx

```diff
@@ -8,58 +9,67 @@ import {
   UIComponent,
   UIComponentProps,
   ChildrenComponentProps,
-  ContentComponentProps,
   commonPropTypes,
+  customPropTypes,
 } from '../../lib'
 import Slot from '../Slot/Slot'
+import { ComponentSlotStylesPrepared } from '../../themes/types'
 
-export interface ChatItemProps
-  extends UIComponentProps,
-    ChildrenComponentProps,
-    ContentComponentProps<ShorthandValue> {}
+export interface ChatItemProps extends UIComponentProps, ChildrenComponentProps {
+  /** Chat items can have a gutter. */
+  gutter?: ShorthandValue
+
+  /** Indicates whether the gutter is positioned at the start or the end. */
+  gutterPosition?: 'start' | 'end'
+
+  /** Chat items can have a message. */
+  message?: ShorthandValue
+}
 
 /**
  * A chat item represents a single event in a chat.
  */
 class ChatItem extends UIComponent<Extendable<ChatItemProps>, any> {
   static className = 'ui-chat__item'
-
   static create: Function
-
   static displayName = 'ChatItem'
 
   static propTypes = {
-    ...commonPropTypes.createCommon({
-      content: 'shorthand',
-    }),
+    ...commonPropTypes.createCommon(),
```

Comment:
Please change the common propTypes to:
```
...commonPropTypes.createCommon({
  content: false,
}),
```

Reply:
I think it's strange having to specify some option to be false, I was expecting to have to specify what I want to include not to exclude. 

Anyway to unblocked this: Since I'm on vacation now, feel free to commit a fix and merge, earliest I have access to PC is tonight.

Or create a github suggestion and I can commit from phone :)

Reply:
I think it's strange having to specify some option to be false, I was expecting to have to specify what I want to include not to exclude. 

Anyway to unblocked this: Since I'm on vacation now, feel free to commit a fix and merge, earliest I have access to PC is tonight.

Or create a github suggestion and I can commit from phone :)

Reply:
I think it's strange having to specify some option to be false, I was expecting to have to specify what I want to include not to exclude. 

Anyway to unblocked this: Since I'm on vacation now, feel free to commit a fix and merge, earliest I have access to PC is tonight.

Or create a github suggestion and I can commit from phone :)

Reply:
More often you want everything, in rear cases you don't want something from the common prop types, that is why we are excluding what we don't want. Otherwise, always when you create common prop types you will have to specify that you want everything... Will commit the change, thanks for info :)

Reply:
```suggestion
    ...commonPropTypes.createCommon({ content: false }),
```